operator/metrics/metrics.go

```diff
@@ -138,6 +165,34 @@ func registerMetrics() []prometheus.Collector {
 	}, []string{LabelOutcome})
 	collectors = append(collectors, IdentityGCRuns)
 
+	CiliumEndpointBatchDensity = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: Namespace,
+		Name:      "number_of_ceps_per_ceb",
+		Help:      "The number of CEPs batched in a CEB",
+	}, []string{})
+	collectors = append(collectors, CiliumEndpointBatchDensity)
+
+	CiliumEndpointsChangeCount = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: Namespace,
+		Name:      "number_of_cep_changes_per_ceb",
+		Help:      "The number of changed CEPs in each CEB update",
+	}, []string{LabelOpcode})
+	collectors = append(collectors, CiliumEndpointsChangeCount)
+
+	CiliumEndpointBatchSyncErrors = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Name:      "ceb_sync_errors_total",
+		Help:      "Number of CEB sync errors",
+	}, []string{})
+	collectors = append(collectors, CiliumEndpointBatchSyncErrors)
+
+	CiliumEndpointBatchQueueDelay = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: Namespace,
+		Name:      "ceb_propagation_delay_seconds",
```

Comment:
name is a bit misleading, this is queueing delay, not propagation delay

Reply:
Done